xbmc/cores/VideoPlayer/DVDCodecs/Overlay/contrib/cc_decoder708.cpp

```diff
@@ -342,8 +342,8 @@ int handle_708_C2 (cc708_service_decoder *decoder, unsigned char *data, int data
 
 int handle_708_C3 (cc708_service_decoder *decoder, unsigned char *data, int data_length)
 {
-  if (data[0]<0x80 || data[0]>0x9F)
-    ;//ccx_common_logging.fatal_ftn (CCX_COMMON_EXIT_BUG_BUG, "Entry in handle_708_C3 with an out of range value.");
+  //if (data[0] < 0x80 || data[0] > 0x9F)
```

Comment:
Hum intentional change? I'd say to remove or keep it. Where is the warning/error exactly? Maybe use Kodi logging instead?

Reply:
since these CC decoder's need to be a bit cleaned and also updated from upstream i have commented code intentionally, but i can add it as you said

Reply:
ah yes the warning was because the "if" has no code inside it end with ";"

Reply:
Hum so maybe log as error in that case